### ISSUE TITLE
In WebStorm, if --ao is put last, it is given the value of the working directory

### ISSUE THREAD

**Issue by lasalvavida** [AnalyticalGraphicsInc/gltf-pipeline#139]
How to reproduce:

1. Make a new Node.js run configuration.
2. run `./bin/gltf-pipeline.js ... --ao`

This actually happens if you put any flag without a value last. But it fails on ao because the typeof check in`bin/gltf-pipeline.js` doesn't work so it just gets assigned to be the path.

**Comment by lasalvavida** [comment]
@likangning93 This has been updated to more accurately reflect the issue

**Comment by likangning93** [comment]
I think this is currently true for any yargs arguments. When I tack something else like `-t` or `--fakeArg` onto the invocation it ends up putting the working directory value with those too.

**Comment by lasalvavida** [comment]
You are correct; I did put that in the last line of the issue. However, I don't think it fails on any of them because we just check if they are defined.

I think just changing:
```javascript
if (typeof(aoOptions) === 'boolean') {
    aoOptions = {};
}
```

to

```javascript
if (typeof(aoOptions) === 'boolean' || typeof(aoOptions) === 'string') {
    aoOptions = {};
}
```

should resolve this.

**Comment by likangning93** [comment]
Ah I see. Thanks!

**Comment by lasalvavida** [comment]
Fixed by #139
